webapp/components/interop-dashboard.js

```diff
@@ -336,54 +338,40 @@ class InteropDashboard extends PolymerElement {
                   <!-- First score table header with sort functionality -->
                   <template is="dom-if" if="[[isFirstTable(itemsIndex)]]">
                     <tr class="section-header">
-                      <th class="sortable-header" on-click="sortByName">
+                      <th class="sortable-header">
                         {{section.name}}
-                        <img class="sort-icon-focus-areas" src="[[getSortIcon(0, sortColumn, isSortedAsc)]]" />
+                        <img class="sort-icon-focus-areas" src="[[getSortIcon(-2, sortColumn, isSortedAsc)]]" />
```

Comment:
It is fine to use a negative index for Focus area column, until we have more types of columns. What if we create a separate function that calls getSortIcon(0,  sortColumn, isSortedAsc) without the negative number hacks?

Reply:
Good idea - I've implemented this!